linux/check_quiesce_snapshot_custom_script/check_quiesce_snapshot_custom_script.yml

```diff
@@ -15,6 +15,13 @@
           vars:
             skip_test_no_vmtools: true
 
+        - name: "Skip if it's not supported"
```

Comment:
Change task name to "Skip test case for FreeBSD". And change when condition with guest_os_ansible_distribution instead of vm_guest_id.